### ISSUE TITLE
Is it possible to add class_choice support to ShapeNetCore.v2 dataset?

### ISSUE THREAD

**Issue by BoodgionWood** [AnTao97/PointCloudDataset#10]
I am trying to load shapenetcorev2_hdf5_2048 dataset. However, I would like only one specific class (for example, chair) in the dataloader. I found that class_choice is not supported for shapenetcorev2, right? Could you please help add class_choice support to ShapeNetCore.v2? Thank you very much!

**Comment by antao97** [comment]
In `dataset.py`, I already define `class_choice`.

**Comment by BoodgionWood** [comment]
May I ask that, for dataset 'shapenetcorev2', what is the form of value you use for class_choice? I tried
 `d = Dataset(root=root, dataset_name='shapenetcorev2',class_choice='chair', num_points=2048, split=split)`
And got return: 

> File "~/dataset.py", line 106, in __init__
    indices = (self.name == class_choice).squeeze()
AttributeError: 'bool' object has no attribute 'squeeze'

I tried parameter class_choice=['chair'] and got the same return.
self.name is a list of class names. self.name=='chair' or self.name==['chair'] would only return one bool value False, which leads to the error.

May I ask what should I write if I want a dataset with only class 'chair'?
